### PR TITLE
Update openai-translator extension

### PR DESCRIPTION
**Pull request by dekura**
## Description

This PR introduces a new feature to the openai-translater that enhances the user experience by automating a common post-translation action. The changes made are as follows:

- `Add preference for auto copy to clipboard`

- `Automatic Clipboard Copy`: Upon successful translation, the translated text is now automatically copied to the clipboard. This saves users the extra step of manually copying the text, making the translation process more seamless and efficient.
- `Updated User Feedback`: The toast message displayed after a successful translation has been updated. The previous message, "Got your translation," has been replaced with "Translation copied to clipboard." This change more accurately reflects the action taken and informs the user that the translated text is ready to be pasted wherever needed.

## Screencast

image

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by douo** [comment]
Looks good to me, I’ll wait for PR #11551  to be merged before I review it. Can you please sync up with this PR first?

**Comment by dekura** [comment]
> Looks good to me, I’ll wait for PR #11551 to be merged before I review it. Can you please sync up with this PR first?

@douo 
Sure,  #11551 merged.

**Comment by pernielsentikaer** [comment]
Awaiting #11551

File: extensions/openai-translator/assets/lang
Comment (douo) [inline extensions/openai-translator/assets/lang:1]: Why changing the permissions?

File: extensions/openai-translator/package.json [hunk 1]
```diff
@@ -620,6 +631,8 @@
     "iso-639-1": "^2.1.15",
     "iso-639-3-to-1": "^1.0.0",
     "node-fetch": "^3.3.0",
+    "ray": "^0.0.1",
```
Comment (douo) [inline extensions/openai-translator/package.json:634]: Is this necessary?

**Comment by dekura** [comment]
@douo What are your thoughts on the pull request now? Can we go ahead and merge it?

**Review by douo — CHANGES_REQUESTED** [review]
@dekura  Please do not touch the binary file, I'm fine with everything else.

File: extensions/openai-translator/package.json (hunk 1 above)
Comment (pernielsentikaer) [reply extensions/openai-translator/package.json:634]: No, that has nothing to do with Raycast

**Comment by pernielsentikaer** [comment]
Could you look into what @dekura mentioned? I compiled the binaries again (I always do that); also, the changelog has a merge conflict 🙂

**Comment by dekura** [comment]
> @dekura Please do not touch the binary file, I'm fine with everything else.

I did not actively modify the binary file. Could it be due to automatic compilation or building? 
I am not quite sure how to solve this issue. Could you please help me resolve it or provide me with some instructions?

**Comment by douo** [comment]
Hi @dekura, please remove binary files from your commit and eliminate any unnecessary dependencies. Additionally, please resolve any conflicts with the mainstream branch.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**